sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -6289,13 +6305,13 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
               = int64_t(); // Specifies the number of bytes being transmitted in the request body.
                            // When the x-ms-write header is set to clear, the value of this header
                            // must be set to zero.
-          Azure::Core::Nullable<std::string>
+          Azure::Core::Nullable<Storage::ContentHash>
               SourceContentCrc64; // Specify the crc64 calculated for the range of bytes that must
                                   // be read from the copy source.
-          Azure::Core::Nullable<std::string>
+          Azure::Core::Nullable<Storage::ContentHash>
               SourceIfMatchCrc64; // Specify the crc64 value to operate only on range with a
                                   // matching crc64 checksum.
-          Azure::Core::Nullable<std::string>
+          Azure::Core::Nullable<Storage::ContentHash>
               SourceIfNoneMatchCrc64; // Specify the crc64 value to operate only on range without a
```

Comment:
I have no idea how we should name these variables.

Reply:
Let's discuss offline.

Reply:
Discussed offline, and those names are not visible for customers.